src/nodes/display/ColorSpaceNode.js

```diff
@@ -93,8 +93,12 @@ export const toWorkingColorSpace = ( node ) => nodeObject( new ColorSpaceNode( n
 export const workingToColorSpace = ( node, colorSpace ) => nodeObject( new ColorSpaceNode( nodeObject( node ), WORKING_COLOR_SPACE, colorSpace ) );
 export const colorSpaceToWorking = ( node, colorSpace ) => nodeObject( new ColorSpaceNode( nodeObject( node ), colorSpace, WORKING_COLOR_SPACE ) );
 
+export const convertColorSpace = ( node, sourceColorSpace, targetColorSpace ) => nodeObject( new ColorSpaceNode( nodeObject( node ), sourceColorSpace, targetColorSpace ) );
+
 addMethodChaining( 'toOutputColorSpace', toOutputColorSpace );
 addMethodChaining( 'toWorkingColorSpace', toWorkingColorSpace );
 
 addMethodChaining( 'workingToColorSpace', workingToColorSpace );
 addMethodChaining( 'colorSpaceToWorking', colorSpaceToWorking );
+
+addMethodChaining( 'convertColorSpace', convertColorSpace );
```

Comment:
I have plans to have the color space information present in the flow of code, that way we will always know what the color space source of the current value. I was thinking about adding a `node.toColorSpace( targetColorSpace )` function in the method chaining.

Maybe we could reverse the order to:

```js
addMethodChaining( 'toColorSpace', ( node ) => convertColorSpace( node, targetColorSpace, sourceColorSpace = null ) );
```

Reply:
>I have plans to have the color space information present in the flow of code ... adding a `node.toColorSpace( targetColorSpace `

That sounds like a good idea to me.

And in addition, consider adding `.colorSpace` as a property of the `Color` class.

Reply:
Both really interesting ideas! I'm not sure of the implications, but this would be good to explore. For now, I've just removed `convertColorSpace` from chaining since it would be a bit confusing to change that chaining API later.